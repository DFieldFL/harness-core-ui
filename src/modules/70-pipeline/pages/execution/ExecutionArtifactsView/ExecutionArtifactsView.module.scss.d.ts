/* eslint-disable */
// this is an auto-generated file
declare const styles: {
  readonly wrapper: string
  readonly emptyArtifacts: string
  readonly stageSelector: string
}
export default styles
