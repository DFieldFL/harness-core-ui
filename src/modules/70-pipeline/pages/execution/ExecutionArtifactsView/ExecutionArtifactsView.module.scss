.wrapper {
  // 96px is ExtendedPageHeader's height
  // 56px is Tabs's height
  // --spacing-huge * 2 is top + bottom margins
  height: calc(100vh - 96px - 56px - var(--spacing-xxxlarge) * 2);
  width: 960px;
  margin-right: auto;
  margin-left: auto;
  overflow: auto;
  margin-top: var(--spacing-xxxlarge);
  margin-bottom: var(--spacing-xxxlarge);
}

.emptyArtifacts {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: var(--spacing-huge) !important;
  p {
    margin-top: var(--spacing-medium) !important;
  }
}

.stageSelector {
  margin-bottom: var(--spacing-medium);
  width: 300px !important;
}
