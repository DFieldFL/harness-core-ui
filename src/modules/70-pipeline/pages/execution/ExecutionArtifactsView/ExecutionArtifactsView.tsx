import React from 'react'

export default function ExecutionArtifactsView(): React.ReactElement {
  return <div>ExecutionArtifactsView</div>
}
