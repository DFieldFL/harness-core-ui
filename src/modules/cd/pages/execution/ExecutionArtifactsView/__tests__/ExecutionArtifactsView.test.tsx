import React from 'react'
import { render } from '@testing-library/react'

import ExecutionArtifactsView from '../ExecutionArtifactsView'

describe('<ExecutionArtifactsView /> tests', () => {
  test('snapshot test', () => {
    const { container } = render(<ExecutionArtifactsView />)
    expect(container).toMatchSnapshot()
  })
})
