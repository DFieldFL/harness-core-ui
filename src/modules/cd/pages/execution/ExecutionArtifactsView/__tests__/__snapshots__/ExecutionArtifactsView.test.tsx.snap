// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExecutionArtifactsView /> tests snapshot test 1`] = `
<div>
  <div>
    ExecutionArtifactsView
  </div>
</div>
`;
